src/JoomlaBrowser.php

```diff
@@ -7,8 +7,10 @@
  */
 
 namespace Codeception\Module;
-
+use Joomla\Browser\Locators\Locators;
 use Codeception\Module\WebDriver;
+require_once('locator/Locators.php');
```

Comment:
you don't need that if you follow namespace directory structure - should be autoloaded ;)